exonum-java-binding/testkit/pom.xml

```diff
@@ -33,11 +33,25 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
```

Comment:
I think an extension must not depend on the aggregate thing, but on `junit-jupiter-api`.